### DISCUSSION TITLE
/cn/2019/02/bezier-curve/

### DISCUSSION THREAD

**Discussion by leovan** [leovan/leovan.me#20] [Comments]
贝塞尔曲线 (Bézier Curve) - Leo Van | 范叶亮 
 https://leovan.me/cn/2019/02/bezier-curve/

**Comment by leovan** [comment]
```header
由 Rick 于 2022-04-11 15:28:57 写道：
````
答主请问的你的字母图案的Bézier Curve的辅助线是怎么画的啊

**Reply by leovan** [reply]
```header
于 2022-04-12 07:06:39 写道：
````
最后两个图是从互联网上找的参考，上面的图都是自己画的，代码在：[https://github.com/leovan/leovan.me/tree/master/scripts/cn/2019-02-19-bezier-curve/bezier-curve-images.py](https://github.com/leovan/leovan.me/tree/master/scripts/cn/2019-02-19-bezier-curve/bezier-curve-images.py)。

**Comment by dagouxiong233** [comment]
好文章
